adapter/internal/ga/client.go

```diff
@@ -239,6 +241,23 @@ func addAPIToChannel(resp *discovery.DiscoveryResponse) {
 			if currentGAAPI.RevisionUUID == api.RevisionUUID {
 				continue
 			}
+			// Related to issue: https://github.com/wso2-enterprise/choreo/issues/14460
+			// This temporary if condition is added to prevent redeploying of all the existing APIs due to change done
+			// in API revision ID to handle environment specific properties. New API revision ID will be in the format
+			// of revisionId_timestamp and the currentGAAPI.RevisionUUID == api.RevisionUUID returns fails for all the
+			// existing APIs. This additional if block is added to prevent redeploying of APIs until a specific timestamp.
+			// Remove this if block after GA is updated with revisionId_timestamp related changes.
+			if len(strings.Split(api.RevisionUUID, "_")) == 2 {
```

Comment:
Shall we do split once and then use that array.

Reply:
fixed